src/components/organisms/Notice/Notice.tsx

```diff
@@ -55,7 +61,7 @@ interface Props {
   dismiss: (id: number) => void;
   confirmDismiss: (id: number) => void;
   undismiss: (id: number) => void;
-  clickContributor: (id: number) => void;
+  clickContributor: (contributor: Contributor) => void;
```

Comment:
Here it would be `onContributorClick` to stay consistent

Reply:
:+1: